src/main/java/org/humancellatlas/ingest/state/persistence/AutoLoader.java

```diff
@@ -103,7 +107,7 @@ public void autoLoad() {
 
     private StateMachine<SubmissionState, SubmissionEvent> updateState(StateMachine<SubmissionState, SubmissionEvent> stateMachine, SubmissionEnvelopeReference envelope) throws UnrecognisedSubmissionStateException {
```

Comment:
Would a more appropriate name would be something like `updateStateMachineToEnvelopeState`?